en/docs/observe-and-manage/working-with-management-api.md

```
@@ -1055,6 +1055,22 @@ The management API has multiple resources to provide information regarding the d
 
 	**Description**: Retrieves information related to a specified task.
 
+### ACTIVATE/DEACTIVATE/TRIGGER TASKS
+
+-	**Resource**: `/tasks`
+
+	**Description**: Activate, deactivate, or tigger a specific task. The status can be set to `active`, `inactive`, or `trigger` to activate, deactivate, or trigger the task, respectively. When a task is triggered, it is executed immediately as a one-time execution.
```

markdownlint-cli2 (0.18.1)

Hard tabs
Column: 1

(MD010, no-hard-tabs)